nucleus/src/main/java/za/co/absa/subatomic/infrastructure/AtomistConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package za.co.absa.subatomic.infrastructure;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Data
+@Configuration
+@ConfigurationProperties("atomist")
+public class AtomistConfiguration {
```

Comment:
Please add a space between class and first property as per current convention.
We need to add formatting rules and Maven plugin to make this a non issue in future.

See #8

Reply:
Until #8 get's implemented, please use the attached [Eclipse Code Formatter](https://plugins.jetbrains.com/plugin/6546-eclipse-code-formatter) file and reformat this PR using it.

The formatting rules can be found in the [`eclipse`](https://github.com/absa-subatomic/gluon/tree/master/eclipse) directory. (Might want to pick up #3 as part of this?)